packages/vercel-edge/tsconfig.test.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+
+  "include": ["test/**/*"],
+
+  "compilerOptions": {
+    // should include all types from `./tsconfig.json` plus types for all test frameworks used
+    "types": ["node", "jest"]
```

Comment:
l: is this the thing that's still left to figure out or can we use `"@edge-runtime/types"` instead of node here?

Reply:
Yup this is part of the todo raised here https://github.com/getsentry/sentry-javascript/blob/1fdca16cb6666ca50d71ffdd0036ee61842cc81d/packages/vercel-edge/jest.config.js#L5-L6